paymentsheet-example/src/androidTest/java/com/stripe/android/test/core/ui/UiAutomatorWebViewText.kt

```diff
@@ -0,0 +1,28 @@
+package com.stripe.android.test.core.ui
+
+import androidx.test.uiautomator.By
+import androidx.test.uiautomator.BySelector
+import androidx.test.uiautomator.UiDevice
+import androidx.test.uiautomator.UiObject2
+import androidx.test.uiautomator.Until
+
+private const val DefaultTimeoutInMillis = 5_000L
+
+open class UiAutomatorWebViewText(
```

Comment:
Adding this class to include a default waiting period to make these webview-based tests more resilient.